Gemfile

```diff
@@ -18,6 +18,7 @@ gem 'hiredis'
 gem 'httparty'
 gem 'lograge'
 gem 'newrelic_rpm'
+gem 'nokogiri' # required for premailer-rails
```

Comment:
This is what the README of the premailer gem suggests: https://github.com/fphilipe/premailer-rails#installation

My guess is that it is currently not working because we don't explicitly have this in the Gemfile, but I could be wrong. Is there a way for me to test this in a prod-like environment before merging?